src/transports/sentry_transport_winhttp.c

```diff
@@ -207,16 +222,18 @@ sentry__winhttp_send_task(void *_envelope, void *_state)
             }
         }
     } else {
-        SENTRY_DEBUGF("http request failed with error: %d", GetLastError());
+        SENTRY_DEBUGF(
+            "`WinHttpSendRequest` failed with code `%d`", GetLastError());
     }
 
+    uint64_t now = sentry__monotonic_time();
+    SENTRY_TRACEF("request handled in %llums", now - started);
+
+exit:
     WinHttpCloseHandle(request);
```

Comment:
Two questions on this:
1. Does this need to be guarded against `NULL`? In examples, I usually see that calls to this method are guarded.
2. In the [WinHttpCloseHandle documentation](https://docs.microsoft.com/en-us/windows/win32/api/winhttp/nf-winhttp-winhttpclosehandle) it says:

> An application should never call WinHttpCloseHandle on a synchronous request. 

Should we actually be calling this? Examples seem to suggest that this is fine, can you just check what this refers to?

Reply:
I think it refers to concurrently calling this on a request that is called in a different thread? All the examples are closing the request.